gp/cartesian_graph.py

```diff
@@ -222,23 +224,19 @@ def _format_output_str_of_all_nodes(self) -> None:
             for node in active_nodes[hidden_column_idx]:
                 node.format_output_str(self)
 
-    def _fill_parameter_values(
-        self, func_str: str, parameter_names_to_values: Optional[Dict[str, float]] = None
-    ) -> str:
+    def _fill_parameter_values(self, func_str: str) -> str:
         g = re.findall("<p[0-9]+>", func_str)
         if len(g) != 0:
-            if parameter_names_to_values is None:
+            if self.parameter_names_to_values is None:
```

Comment:
not sure whether this if branch is needed, will investigate tomorrow

Reply:
what's the outcome here?

Reply:
not needed